source/dbo/Views/Power BI Reporting/vwApplicationUsage.sql

```diff
@@ -0,0 +1,22 @@
+﻿CREATE VIEW [dbo].[vwApplicationUsage]
+	AS
+
+	WITH
+	Usage AS 
+	(
+		SELECT c.ISOYearWeek AS YearWeek, SUM(Q1.NumberOfRows) AS NumberOfAudits
+		FROM [Calendar] c
+			LEFT OUTER JOIN 
+			(SELECT CONVERT(DATE, [AuditDateTime]) AS AuditDate, COUNT(iD) AS NumberOfRows
+			FROM [$(NTBS_AUDIT)].[dbo].[AuditLogs]
+			WHERE AuditUser != 'SYSTEM'
+			GROUP BY CONVERT(DATE, [AuditDateTime])) as q1 on q1.AuditDate = c.DateValue
+			WHERE c.DateValue between '2021-07-10' and GETUTCDATE()
```

Comment:
There's a bit of a mix of case here. I know SQL doesn't mind about case but we should try to be consistent. I usually will use uppercase for keywords like you have with `GROUP BY` and such. So `as`, `on` and `and` should all be uppercased.